man/org.freedesktop.systemd1.xml

```diff
@@ -265,6 +265,7 @@ node /org/freedesktop/systemd1 {
       LookupDynamicUserByUID(in  u uid,
                              out s name);
       GetDynamicUsers(out a(us) users);
+      GetIdentityMapping(out h namespace);
```

Comment:
maybe GetIdentityUserNamespace()?

Reply:
that's a bit of a mouthful - that you get a namespace is encoded in the output parameter so to me it seems clearer this way